### PR TITLE
Truncating data #11

### LINKED ISSUE #11: Truncating data for fields
Fields `title`, `slug`, `description`, `thumbnail` are truncated after save.
I'm using MySQL.

Please, check field types.

### PR DESCRIPTION
**Pull request by Madlexx**
close #11

### CODE REVIEW

File: app/config/config.yml [hunk 1]
```diff
@@ -86,3 +86,8 @@ fos_user:
 
 application_blog:
     upload_path: '/upload/my_thumbnails'
+
+stof_doctrine_extensions:
+    orm:
+        default:
+             sluggable: true
```
Comment (aivus) [inline app/config/config.yml]: Don't forget about new line

File: src/Application/BlogBundle/Admin/CategoryAdmin.php [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 namespace Application\BlogBundle\Admin;
 
+use Application\BlogBundle\Form\Type\ThumbnailType;
```
Comment (aivus) [inline src/Application/BlogBundle/Admin/CategoryAdmin.php]: Group it, please

Comment (Madlexx) [inline src/Application/BlogBundle/Admin/CategoryAdmin.php]: Fixed.

**Comment by aivus** [comment]
Please, do backmerge

**Pull request closed without merging**